_rules/html-page-lang-b5c3f8.md

```diff
@@ -61,31 +61,31 @@ The `lang` attribute specified has a non-empty value.
 
 #### Failed Example 1
 
-There is no `lang` attribute specified.
+This `html` element does not have a `lang` attribute.
 
 ```html
 <html></html>
 ```
 
 #### Failed Example 2
 
-The `lang` attribute specified is empty ("").
+This `html` element has a `lang` attribute with an empty (`""`) value.
```

Comment:
```suggestion
This `html` element should not have a `lang` attribute with an empty (`""`) value.
```

Reply:
I think this is a good example of why "should" is maybe not as clear. Is this failing because the attribute is empty or because it should have an empty attribute.

It also could lead to double negative confusion, and Jey indicated it could be an issue for unit test stuff.